app/assets/javascripts/select-kit/addon/components/bulk-select-bookmarks-dropdown.js

```diff
@@ -0,0 +1,113 @@
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import Bookmark from "discourse/models/bookmark";
+import i18n from "discourse-common/helpers/i18n";
+import DropdownSelectBoxComponent from "select-kit/components/dropdown-select-box";
+
+const _customButtons = [];
+const _customActions = {};
+
+export function addBulkDropdownAction(name, customAction) {
+  _customActions[name] = customAction;
+}
+
+export function addBulkDropdownButton(opts) {
+  _customButtons.push({
+    id: opts.label,
+    icon: opts.icon,
+    name: i18n(opts.label),
+    visible: opts.visible,
+  });
+  addBulkDropdownAction(opts.label, opts.action);
+}
+
+export default DropdownSelectBoxComponent.extend({
+  classNames: ["bulk-select-bookmarks-dropdown"],
+  headerIcon: null,
+  showFullTitle: true,
+  selectKitOptions: {
+    showCaret: true,
+    showFullTitle: true,
+    none: "select_kit.components.bulk_select_bookmarks_dropdown.title",
+  },
+
+  router: service(),
+  toasts: service(),
+  dialog: service(),
+
+  computeContent() {
```

Comment:
Can you try using the `content: ` property? `computeContent` is marked as deprecated, although, we seem to be silencing those deprecations. I am curious if actually switching to `content: ` will address the test issues as well.

Reply:
Unfortunately, this update didn't help the test issue. I still see the error sometimes if I add multiple tests 🤔

Reply:
Yeah, quiet strange. I think we can let this be for now.